frontend/src/routes/ApplicationForm/FormStructure.jsx

```diff
@@ -82,9 +80,9 @@ const FormStructure = ({
             Søknaden ble sist registrert
             {myApplication && (
               <StyledSpan bold>
-                <Moment format="dddd Do MMMM, \k\l. HH:mm:ss">
+                <FormatTime format="EEEE dd MMMM, kl. HH:mm:ss">
```

Comment:
Could this be changed from `dd` to `d.`? The `Do` format in moment seems to automatically add the dot, we didn't have padded date strings before and personally I prefer `4.` over `04`

Reply:
ah, yeah sure